### PR TITLE
Imagemapper lut range

### PR DESCRIPTION
**Pull request by sankhesh**
### Context

This change adds functionality to map pixel data to colors using one of two mechanisms in the image
mapper:

- Use color window/level values from the image property
- Use the range set on the lookup table in the image property

By default, the color window/level values are used.

BREAKING CHANGE: The original behavior of the image mapper was that if a lookup table is provided,
it mapped the lookup table's scalar range by default. The new behavior disables using the lookup
table scalar range by default. Instead, the window/level values are used.
### Changes

- [x] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing

- [x] This change adds or fixes unit tests 
- [x] Tested environment:
  - **vtk.js**:  latest master
  - **OS**:  Linux
  - **Browser**:  Brave, Firefox

### CODE REVIEW

**Comment by jourdain** [comment]
If that breaking change makes sense to the medical folks and @martinken, I'm cool with it.

File: Sources/Rendering/Core/ImageProperty/index.d.ts [hunk 1]
```diff
@@ -173,6 +174,12 @@ export interface vtkImageProperty extends vtkObject {
 	 * @param func 
 	 */
 	setScalarOpacity(index: any, func: any): boolean;
+
+	/**
+	 *
```
Comment (finetjul) [inline Sources/Rendering/Core/ImageProperty/index.d.ts:179]: please add documentation

**Review by finetjul — COMMENTED** [review]
LGTM, it is the same behavior as in VTK C++

**Review by martinken — APPROVED** [review]

File: Sources/Rendering/Core/ImageProperty/index.d.ts (hunk 1 above)
Comment (sankhesh) [reply Sources/Rendering/Core/ImageProperty/index.d.ts:179]: Thanks, will do.

**Comment by osiux** [comment]
> Nope. Not when a LUT is supplied. When a lut is supplied, the lut is range was used and there was no mechanism to window/level the range.

What is LUT? The lookup table? Does this mean that if this is used, is not possible to change window/level?

**Comment by martinken** [comment]
Ahh yes I see. I keep thinking we do W/L in the fragment shader and we don't yet. LGTM

**Pull request merged**